### PR TITLE
fixing docstring of read_block function

### PR DESCRIPTION
**Pull request by evanfwelch**
I am in the middle of debugging an authetntication to S3 error and just found it

### CODE REVIEW

**Comment by martindurant** [comment]
You mean that `f` in the function is an open file, not a file-name to be opened?

File: distributed/utils.py [hunk 1]
```diff
@@ -611,7 +611,7 @@ def read_block(f, offset, length, delimiter=None):
 
     Parameters
     ----------
-    fn: string
+    f: string
```
Comment (mrocklin) [inline distributed/utils.py]: `f` is commonly used for a file while `fn` is commonly used for a filename.  If the intent is for this to be a filename then we might instead change f->fn everywhere.  If this is commonly used for non-file paths then we might instead use the term `path` everywhere.

@martindurant any preferences?

Comment (martindurant) [inline distributed/utils.py]: In the code that follows, it is definitely a file (not a string), so `f`. I suspect that in a previous version, it was indeed a string, and this is vestigial.

**Comment by evanfwelch** [comment]
Thanks for your attention here, I should have been more clear. Yeah f is
definitely a file in this function so just fixing the docstring to reflect
that. I like the f vs fn convention and have noticed it elsewhere.

**Comment by mrocklin** [comment]
I've updated the docstring to point out that this is a file, and not a string.  Merging on passed tests.

**Pull request merged**